doc/sphinxext/sphinx_gallery/_static/gallery.css

```diff
@@ -209,7 +209,6 @@ a.sphx-glr-code-links[tooltip]:hover:before{
     display: inline-block;
     text-align: center;
     text-indent: 0;
-    margin-left: -5em;
+    margin-left: 0; /* Has to be zero to avoid overlapping with sidebar */
```

Comment:
It is a modification done in place for now for scikit learn. Which I believe to be reasonable enough to  transfer to sphinx-gallery.
Code tooltips are a recent feature in Sphinx-Gallery from your last PR as you wanted to make their use more explicit for Scipy-lectures code examples.
From a personal point of view, I see little advantage on trying to make them look a bit centered relative to the link and left aligning seems good enough. I was once slightly annoyed  they apear bellow the link, because if 2 consecutive lines have function calls, the first tooltip will block the second link and you have to move the mouse around to make them disappear and select the second function link. Then I thought making them inline would be better, and then again I thought that a nested function call I would experience the same issue, so I let it be.